src/components/Navigation/Navigation.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
```

Comment:
I noticed there are semi colons on some imports and not on others. We should decide on a convention to use

Reply:
Good point, we can decide today